BUILD.bazel

```diff
@@ -11,10 +13,24 @@ exports_files([
     "tsconfig.json",  # @external
     "tsconfig-test.json",  # @external
     "tsconfig-build.json",  # @external
+    "package.json",
 ])
 
 # Detect if the build is running under --stamp
 config_setting(
     name = "stamp",
     values = {"stamp": "true"},
 )
+
+# If set will replace dependency versions with tarballs for packages in this repo
+bool_flag(
+    name = "enable_package_json_tar_deps",
+    build_setting_default = False,
+)
+
+config_setting(
+    name = "package_json_use_tar_deps",
+    flag_values = {
+        ":enable_package_json_tar_deps": "true",
+    },
+)
```

Comment:
This is enabled by running `yarn build:bazel --local`. Once we flip to using the bazel script it will just be `yarn build --local`.